CHANGES.md

```diff
@@ -24,6 +24,10 @@ OpenSSL 3.2
 
 ### Changes between 3.0 and 3.2 [xx XXX xxxx]
 
+ * Added support for provider-based TLS signature algorithms
```

Comment:
Could we add the word "pluggable" here? It's the way we described the equivalent feature for the TLS key exchange/agreement. e.g. "Added support for pluggable provider-based TLS signature algorithms"

Reply:
So done in https://github.com/openssl/openssl/pull/19312/commits/8be6985a9743a239684308b6258691d3489e52bf